src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -548,92 +548,4 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
 
     return null;
   }
-
-  /**
-   * A {@link FlutterSdk} that is compatible with the Bazel version of the Dart and Flutter SDK.
-   */
-  public static class BazelSdk extends FlutterSdk {
```

Comment:
This class wasn't ever used and doesn't seem compatible with current implementations.